bgpd/bgp_fsm.c

```diff
@@ -1566,6 +1652,91 @@ static int bgp_fsm_holdtime_expire(struct peer *peer)
 	return bgp_stop_with_notify(peer, BGP_NOTIFY_HOLD_ERR, 0);
 }
 
+/* Start the selection deferral timer thread for the specified AFI, SAFI */
+static int bgp_start_deferral_timer(struct bgp *bgp, afi_t afi, safi_t safi,
```

Comment:
since this is 0 and 1, can be converted to bool?

Reply:
not exactly it can have one of the three values 0,1 & -1